### DISCUSSION TITLE
CHARMAPにおける変換元の複数文字のサポート

### DISCUSSION THREAD

**Discussion by SuperTurboZ** [AILight/AILZ80ASM#206] [Ideas]
CHARMAP は他のアセンブラには無い特徴で便利な機能だと思います。
XORの単純な暗号文字列や、独自のコードセットを持つプログラム用の文字データの生成に使わせていただいています。

変換元は１文字という制限がありますが、これを無くせるともっと便利になるかなと思いました。
登録順に優先順位がつけられるであれば、こんな感じにタグ定義ができると応用範囲が広がりそうです。

```
{
"A": [ 65 ],
"B": [ 66 ],
"C": [ 67 ],
"{": [ 123 ],
"}": [ 125 ],
"{CR}": [ 13 ],
"{ESC}": [ 33 ]
}
```

**Comment by AILight** [comment]
ご提案有難うございます。面白いですね。ちょっとこれは組み込めるか確認します！！

**Comment by AILight** [comment]
上記のコードが書けるように修正しました。次回リリースで対応されます。

**Comment by AILight** [comment]
対応版をリリースしました。
